src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -29,7 +29,21 @@
   {{ form_start(currencyForm, {
     'attr': {
       'id': 'currency_form',
-      'data-get-cldr-data': path('admin_currencies_get_reference_data', {'currencyIsoCode': 'CURRENCY_ISO_CODE'}),
+      'data-reference-url': path('admin_currencies_get_reference_data', {'currencyIsoCode': 'CURRENCY_ISO_CODE'})|replace({'/CURRENCY_ISO_CODE': '{/id}'}),
+      'data-languages': languages|json_encode|replace("'", "&#39;")|raw,
+      'data-translations': {
+        '1. Enter symbol': '1. Enter symbol'|trans({}, 'Admin.International.Feature'),
+        '2. Choose format': '2. Choose format'|trans({}, 'Admin.International.Feature'),
+        'Customize symbol and format': 'Customize symbol and format'|trans({}, 'Admin.International.Feature'),
+        'Apply': 'Apply'|trans({}, 'Admin.Actions'),
+        'Cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+        'Close': 'Close'|trans({}, 'Admin.Actions'),
```

Comment:
Do we need to have two buttons for the same action? Cancel and close mean the same to me, I'd keep only the 'Cancel' button and delete the 'Close' action. :-)